### PR TITLE
Clarify GCP Quota Increases

### LINKED ISSUE #800: Clarity on how to increase GCP quotas
**Affected Document:**

Google Cloud Install Procedure

**Problem Type (Priority):**

- [ ] Error (Critical): This document has existing information that's incorrect and prevents readers from completing the documented setup (Add the `error/critical` label to this issue)
- [x] Misleading (Important): This document could benefit from additional context, guidance, or clarity (Add the `error/FYI` label to this issue)
- [ ] Typo (Low): Any issue related to spelling/ punctuation/ formatting (Add the `error/typo` label to this issue)

**Suggested Change (include links to relevant existing docs, support tickets, messages, or external content):**

Customer was confused about how to increase quota (https://astronomer.zendesk.com/agent/tickets/10249) may need to make this section more descriptive.

**Additional Notes:**

### PR DESCRIPTION
**Pull request by kwcanuck**
Resolves #800 

@stuart23 as @jwitz recommended, I added a link to the Google doc [Managing your quota using the Cloud console](https://cloud.google.com/docs/quota#managing_your_quota_console) in the **Prerequisites** section of  [Install Astro on GCP](https://docs.astronomer.io/astro/install-gcp).  The link is preceded by the text _To adjust your project's quota limits up or down_. Hopefully, this alleviates the concern you identified in the issue. When time permits, please review this change and provide your edits or approval. Thanks!

### CODE REVIEW

File: astro/install-gcp.md [hunk 1]
```diff
@@ -7,28 +7,26 @@ description: Get started on Astro by creating your first Astro Cluster on Google
...
+- A [Google Cloud project](https://cloud.google.com/resource-manager/docs/creating-managing-projects) with billing enabled. For security reasons, the install process is not currently supported on a Google Cloud project that has other tooling running in it.
 - A user with [Owner permissions](https://cloud.google.com/iam/docs/understanding-roles) in your project.
 - [Google Cloud Shell](https://cloud.google.com/shell).
 - A minimum [CPU](https://cloud.google.com/compute/quotas#cpu_quota) quota of 36.
-- A minimum [N2_CPU](https://cloud.google.com/compute/quotas#cpu_quota) quota of 24.
-- A subscription to the [Astro Status Page](https://status.astronomer.io). This will ensure that you're alerted in the case of an incident or scheduled maintenance.
+- A minimum [N2_CPU](https://cloud.google.com/compute/quotas#cpu_quota) quota of 24. To adjust your project's quota limits up or down, see [Managing your quota using the Cloud console](https://cloud.google.com/docs/quota#managing_your_quota_console).
```
Comment (jwitz) [inline astro/install-gcp.md:28]: Should we duplicate this link for the prior prerequisite or combine the two quota prerequisites together? Just to make sure that people see we're speaking about quotas in the same context.

File: astro/install-gcp.md [hunk 2]
```diff
@@ -7,28 +7,26 @@ description: Get started on Astro by creating your first Astro Cluster on Google
...
-This guide provides steps for getting started with Astro on Google Cloud Platform (GCP). Below, you'll find instructions for how to complete the Astro install process, including prerequisites and the steps required for our team to provision resources in your network.
+This is where you'll find instructions for installing Astro on the Google Cloud Platform (GCP), including prerequisites and the steps required to allow Astronomer to provision your network resources.
 
-At a high-level, we'll ask that you come prepared with a new Google Cloud project. From there, you can expect to:
+To complete the installation process, you'll:
 
 - Create an account on Astro.
 - Activate your Astro Data Plane by enabling Google Cloud APIs and adding service accounts to your project's IAM.
-- Share information about your Google Cloud project with our team.
+- Share information about your Google Cloud project with Astronomer.
```
Comment (jwitz) [inline astro/install-gcp.md:16]:
```suggestion
- Share information about your Google Cloud project with Astronomer.
- Let Astronomer complete the install.
```

**Review by jwitz — APPROVED** [review]
Few minor comments but LGTM otherwise!

File: astro/install-gcp.md (hunk 1 above)
Comment (stuart23) [reply astro/install-gcp.md:28]: This bit is the crux of the change. Should we just call it out in an info box?

Comment (kwcanuck) [reply astro/install-gcp.md:28]: @stuart23 and @jwitz I duplicated the link. I prefer this method over adding an info box. IF you disagree, reopen this PR. Thanks!

File: astro/install-gcp.md (hunk 2 above)
Comment (kwcanuck) [reply astro/install-gcp.md:16]: @jwitz I don't think this addition is necessary as this statement follows the bulleted list:

_When you've completed the installation process, Astronomer will create a Cluster within your Google Cloud project to host the resources and Apache Airflow components necessary to deploy DAGs and execute tasks._

I think it's already clear that Astronomer will be completing the install. If you disagree, you can reopen the PR.

**Pull request merged**